assets/templates/waitlist.html.tmpl

```diff
@@ -8,11 +8,26 @@
 {{ template "header.html.tmpl" .}}
 </head>
 <script type='text/javascript'>
-//Function to disable the buzz button once clicked.
+
 $(document).ready(function () {
+  //Function to disable the buzz button once clicked.
   $('#buzzid').click(function() {
      $(this).attr('disabled', true);
   });
+
+  //Function from Buzzer.js to update the party size when changed in dropdown in table.
+  //This is necessary here because without this, you must wait for page to refresh before 
+  //you can change party size. If there you know a cleaner way to do this, please inform Kev, Jr. Bootstrap Expert
+    $(".dropdown li a").click(function(){
```

Comment:
Once you rename the register click handler function to `registerUpdatePartySizeClickHandler`, add it to the `$(document).ready(function() {` body along with the other initial click handler registrations. That should fix this problem. As a general rule there shouldn't be any js in the template files. Not sure what the buzz button disabling is doing in here, you can get rid of it because I'm pretty sure that's already being handled in the buzz button click handler in `buzzer.js`